aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -488,8 +488,8 @@ module aptos_framework::voting {
     }
 
     public fun is_voting_closed<ProposalType: store>(voting_forum_address: address, proposal_id: u64): bool acquires VotingForum {
-        let voting_forum = borrow_global_mut<VotingForum<ProposalType>>(voting_forum_address);
-        let proposal = table::borrow_mut(&mut voting_forum.proposals, proposal_id);
+        let voting_forum = borrow_global<VotingForum<ProposalType>>(voting_forum_address);
```

Comment:
This will change the compiled bytecode although it should not change the behavior. Will it be ok? @0xchloe , @movekevin

Reply:
yeah i think it's fine, wdyt @movekevin?

Reply:
Yeah I think this is fine and can be classified as minor no-op changes.